### DISCUSSION TITLE
Plugin installation.

### DISCUSSION THREAD

**Discussion by yetanothern** [apiman/apiman#1854] [Q&A] [answered]
Is it possible to install apiman plugins on instance behind proxy?
I've added JAVA_OPTS="$JAVA_OPTS -Dhttp.proxyHost=ip -Dhttp.proxyPort=port -Dhttp.nonProxyHosts=localhost\|127.0.0.1" into bin/standalone.conf of wildfly instnace and still got an error "WARN: plugin registry failed to load - https://cdn.jsdelivr.net/gh/apiman/apiman-plugin-registry@master/registry.json" while accessing Administration/Manage Plugins in apimanui.

Is there a way to add plugin manually?

**Comment by volkflo** [comment]
Normally Apiman should pick up the proxy settings (also for the plugins).
Can you provide us the error logs/stacktraces from the logs when this error appears?

**Comment by yetanothern** [comment, marked as answer]
Was able to resolve this with local nexus mirror and placing plugin catalogue locally.

-  download file https://cdn.jsdelivr.net/gh/apiman/apiman-plugin-registry@master/registry.json
- place it to standalone/configuration dir
- make a mirror on nexus for https://repo1.maven.org/maven2/
- add into `standalone/configuration/apiman.properties`:
```
apiman.plugins.repositories=http://your-local-nexus/repository/your-name-of-maven-central-mirror/
apiman-manager.plugins.registries=file:///${jboss.server.config.dir}/registry.json
```
